mappings/net/minecraft/world/biome/source/BiomeAccessStrategy.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_4545 net/minecraft/world/biome/source/BiomeAccessStrategy
```

Comment:
What does this class do in code exactly?

Reply:
> For 4545, I named it a `BiomeAccessStrategy`. It can be defined by dimension types. It accepts the world seed, block xyz, and 4544 (the storage) and returns a specific biome. The default impl (4546) has some fuzzy logic that maybe prevents the biome edges look too blocky (as vanilla biomes are stored in 444 units now). There is also an alternative impl (4547) that ignores the y position and delegates to the default impl, which is currently used by the overworld dimension. I named 4546 `DefaultBiomeAccessStrategy` and 4547 `HorizontalBiomeAccessStrategy` as 4547 only cares about the horizontal plane.

It accepts the world seed, block xyz, and 4544 (the storage) and returns a specific biome.

Reply:
The class could potentially just be called `BiomeAccess`

Reply:
Yeah, I would drop the `Strategy`

Reply:
There is already an accessor. Imo the two class names would sound too similar

Reply:
There is already an accessor. Imo the two class names would sound too similar

Reply:
Yeah, now there is `BiomeAccess` and `BiomeAccessType` instead